web/fido_app/templates/profile.html

```diff
@@ -24,6 +24,8 @@
 <canvas width="400" height="400" id="snake-game" class="card"></canvas>
 
 <footer>
+  <a id="add-credential" class="btn btn-outline-primary btn-block profile-action {% if current_user.password_hash %}disabled{% endif %}" href="/add-password">Add Password</a>
+  <a id="add-biometric" class="btn btn-outline-primary btn-block profile-action {% if current_user.credential_id %}disabled{% endif %}" href="/webauthn/registration/add-start">Add Biometric</a>
```

Comment:
Unfortunately, you cannot use `<a>` tags to make `POST` requests (HTML is lame like that 😢).

The workaround to do the following:  
```html
<form method="POST" action="/your/api/route">
   <button type="submit" ...>Link Text</button>
</form>`
```
If the form needs hidden form fields (with known values), you can add those using `<input type="hidden">`.

You'd only need to do this for the webauthn route